h/services/group_links.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+
+class GroupLinksService(object):
+
+    """
+    A service for providing appropriate links (URLs) for a given group object
+    """
+
+    def __init__(self, request_authority, route_url):
+        """
+        Create a group_links service.
+
+        :param request_authority: The request's "default" authority
```

Comment:
As discussed on Slack, the `request.authority` property is really just a setting of the h service as a whole and has nothing to do with the context of the request. Perhaps name this `default_authority` and change the description to "h's default authority"?